CHANGELOG.md

```diff
@@ -6,6 +6,15 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+### Breaking Changes
```

Comment:
I'm calling the Node upgrade a breaking change. I think it is because if you upgrade to the Node 14 packages we don't want to guarantee compatibility with Node 12. Does that sound reasonable? This is debatable.